Developer-Guide.md

```diff
@@ -517,12 +517,14 @@ $ sudo sed -i '$a Requires=kata-debug.service' ${ROOTFS_DIR}/lib/systemd/system/
 ### Build the debug image
 
 Follow the instructions in the [Build a rootfs image](#build-a-rootfs-image)
-section.
+section when using rootfs, or when using initrd, complete the steps in the [Build an initrd image](#build-an-initrd-image) section.
 
 ### Configure runtime for custom debug image
 
 Install the image:
 
+**Note**: When using an initrd image, replace the below rootfs image name `kata-containers.img` with the initrd image name `kata-containers-initrd.img`.
```

Comment:
This needs a leading `>` so that it render as:

> **Note**: When using an ...

See https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes.

Reply:
Thanks @jodh-intel . I've made the required change.